src/foundation.cs

```diff
@@ -9827,28 +9884,35 @@ partial interface NSBundle {
 
 		// Additions from AppKit
 #if MONOMAC
```

Comment:
Can this `#if MONOMAC` be removed?

Reply:
I'll give this one a shot. I don't see why not.

Reply:
Done!